sdk/python/kfp/dsl/_component_bridge.py

```diff
@@ -42,6 +42,8 @@ def _create_container_op_from_component_and_arguments(
 
     container_spec = component_spec.implementation.container
 
+    old_warn_value = dsl.DISABLE_REUSABLE_COMPONENT_WARNING
+    DISABLE_REUSABLE_COMPONENT_WARNING = True
```

Comment:
might be a stupid question

python is single threaded? any chance this can be a problem? like multithreading and recursion?

also, should this line be `dsl.DISABLE_REUSABLE_COMPONENT_WARNING = True`?

Reply:
Thanks for catching this.
Fixed.

I've added tests to verify that the warnings are shown/not shown.